### PR TITLE
Undo changes since Feb. 13

### PR DESCRIPTION
**Pull request by ajordanBBN**

### CODE REVIEW

**Comment by ajordanBBN** [comment]
Method:

```
git checkout -b revert-to-feb13
git checkout 4d0e27e5819aafb7faecf43393a8c36ec84119c0 .
git commit -m "Undo changes since Feb 13"
git push --set-upstream origin revert-to-feb13
```

Then open this PR.

File: virtue/virtue-base/sshd_config [hunk 1]
```diff
@@ -62,7 +62,6 @@ PasswordAuthentication no
 #GSSAPICleanupCredentials yes
 
 X11Forwarding yes
-X11UseLocalhost no
```
Comment (lilk-bit) [inline virtue/virtue-base/sshd_config:65]: What commit did this come from? This wasn't a change made related to resources.

Comment (ajordanBBN) [reply virtue/virtue-base/sshd_config:65]: I made the change, as it allowed X11 forwarding to work better when I was debugging

**Review by fpatwa — APPROVED** [review]
Looks good.

**Pull request merged**